lib/moveTechniquesToExtension.js

```diff
@@ -60,27 +61,35 @@ function moveTechniquesToExtension(gltf) {
                 };
 
                 // Store the name of the uniform to update material values.
-                mappedUniforms[parameterName] = uniformName;
+                if (!defined(mappedUniforms[techniqueId])) {
+                    mappedUniforms[techniqueId] = {};
+                }
+                mappedUniforms[techniqueId][parameterName] = uniformName;
             });
 
-            const programLegacy = gltf.programs[techniqueLegacy.program];
-            const program = {
-                name: programLegacy.name,
-                fragmentShader: undefined,
-                vertexShader: undefined,
-                glExtensions: glExtensions
-            };
+            if (!defined(seenPrograms[techniqueLegacy.program])) {
```

Comment:
Does this need to set `techniqueLegacy.program` in an `else` block?

Reply:
Thanks, fixed!